src/permitted-handler.d.ts

```diff
@@ -0,0 +1,49 @@
+import type {
+  JsonRpcEngineEndCallback,
+  JsonRpcEngineNextCallback,
+  JsonRpcRequest,
+  PendingJsonRpcResponse,
+} from './json-rpc';
+
+/**
+ * Makes every specified property of the specified object type mutable.
+ *
+ * @template T - The object whose readonly properties to make mutable.
+ * @template TargetKey - The property key(s) to make mutable.
+ */
+export type Mutable<
```

Comment:
This type doesn't seem to have much to do with the `PermittedHandlerExport` type. Is there a reason it's exported in this file? Could it be moved to a separate file instead?

Reply:
No reason, just an oversight when moving it over. Have moved it now